dev-docs/add-video-support-to-an-existing-bidder-adaptor.md

```diff
@@ -1,82 +1,10 @@
 ---
+redirect_to: "/dev-docs/bidder-adapter.html"
```

Comment:
These redirects appear broken for me.  It looks like the page that exists is `/dev-docs/bidder-adaptor.html`.  Can we rename that page to `/dev-docs/bidder-adapter.html`?

Reply:
Addressed in d75cfe9

I fixed the busted redirects, and added a page `bidder-adapter` (with an 'e') so anyone using that spelling will land at the main page.

I don't want to rename the existing page because of the risk of breaking existing links/bookmarks and losing the current Google ranking for that URL.